Scripts/Swift/iOSTemplateMaker/Sources/iOSTemplateMaker/SetUpCICDService.swift

```diff
@@ -48,59 +61,62 @@ struct SetUpCICDService {
                 return nil
             }
         }
+
+        var title: String {
+            switch self {
+            case .macOSLatest:
+                "macos"
```

Comment:
the value here should be `macos-latest`. There are another options like `macos-14`